### DISCUSSION TITLE
AIdeck poor WiFi signal

### DISCUSSION THREAD

**Discussion by mfxjx000205** [bitcraze/discussions#882] [Q&A] [answered]
Hi,everyone:

I'm using AIdeck for my attempts at WiFi transferring data. I followed the instructions in **WiFi-img-streamer** exactly: 1. updating crazyflie's firmware via cfclient; 2. flashing aideck's bootloader on a virtual machine; 3. flashing the wifi example using cfloader. After that, it works fine.

However, the most notable issue is that the AIdeck only has a very weak WiFi signal. When I use my phone to connect to AIdeck's hotspot, the signal disappears about 2 meters away.And when I use the computer is, the distance where the signal disappears is even shorter to one meter.

I would like to ask the following questions:
1. am I doing the steps correctly and do I need to do anything other than the three steps above. I am using AIdeck 1.1.
2. I tried to adjust the RF transmit power in the menuconfig of esp-idf and now it is already max 20dbM. is there any limitations related to this in the code of bitcraze.
3. In your opinion what is the most likely cause? I have tried about three AIdeck and the ranges are all equally short.

Best wishes.

**Comment by krichardsson** [comment, marked as answer]
This sound a bit odd, you should not need to do any special settings for the wifi and it should reach more than 1 or 2 meters with the demo.
One thing that could cause this type of problems is if the antenna is broken off. Does your antenna look OK? You can compare with the image here https://www.bitcraze.io/products/ai-deck/. It sounds unlikely though if you have tried with 3 different decks.

**Comment by mfxjx000205** [comment]
Hi，
After my subsequent attempts to reflash the esp firmware and replace the drone, everything worked just fine.
It seems to me that the lack of voltage to the drone could be the cause.
thx for your answer!
